pkg/errors/errors.go

```diff
@@ -116,6 +117,8 @@ func GetErrorCode(err error) string {
 		return "Delete Failed"
 	} else if IsDeletePending(err) {
 		return "Delete Pending"
+	} else if IsAttachmentFailed(err) {
+		return "Attachment Failed"
```

Comment:
Could you raise the moc-pkg PR as well? I would like to see how this error is being used. Wondering if we can make this error more generic.

Reply:
Sure